ci/release/update-version.sh

```diff
@@ -51,3 +51,10 @@ sed_runner 's/'"CUDA_DATAFRAME VERSION .* LANGUAGES"'/'"CUDA_DATAFRAME VERSION $
 # RTD update
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/cudf/source/conf.py
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/nvstrings/source/conf.py
+
+# bump rmm
+CURRENT_RMM="rmm=${CURRENT_MAJOR}.${CURRENT_MINOR}.*"
+NEXT_RMM="rmm=${NEXT_SHORT_TAG}.*"
+for ENV in conda/environments/*.yml; do
+  sed_runner "s/${CURRENT_RMM}/${NEXT_RMM}/g" ${ENV};
```

Comment:
In reference to the above change, this would be changed to

`sed_runner "s/rmm=${CURRENT_SHORT_TAG}.*/rmm=${NEXT_SHORT_TAG}.*/g" ${ENV};`

Reply:
Also maybe change `ENV` to `FILE` just for sensibility